wifiphisher/macmatcher.py

```diff
@@ -0,0 +1,45 @@
+import os
+from constants import *
```

Comment:
I believe he got this from `pywifiphisher.py`. It is fine as the constants are always in capitals.